network_modeling/configs/templates/1.3/aruba/common/sw-cdu.primary.j2

```diff
@@ -47,37 +79,66 @@ interface mgmt
 
 interface loopback 0
     ip address {{ variables.LOOPBACK_IP }}/32
+    ipv6 address  {{ variables.LOOPBACK_IPv6 }}/128
     ip ospf 1 area 0.0.0.0
+    ipv6 ospfv3 1 area 0.0.0.0
+interface loopback 1
+    ipv6 address  {{ variables.LOOPBACK_IPv6 }}/128
+    ipv6 ospfv3 2 area 0.0.0.0
 interface vlan 1
     ip mtu 9198
     ip address {{ variables.MTL_IP }}/{{variables.MTL_PREFIX_LEN}}
+    ipv6 address {{ variables.MTL_IPv6 }}/{{variables.MTL_PREFIX_LEN_IPv6}}
```

Comment:
We need a comprehensive ticket (MTL or CASMNET) to add IPv6 to CSI. The IPv6 doc/worksheet you are creating should be reference or attached to this ticket so that whoever picks up the ticket can implement appropriately.

Reply:
I have SLS CSI ticket already created in which this would be addressed.

I am tracking all newly created variables in a doc that I am creating for IPv6. 

Tickets below:

Phase 6: Document initial IPv6 design (docs-csm)

https://jira-pro.its.hpecorp.net:8443/browse/CASMNET-1647

Phase 10: Ipv6 tooling updates for SLS CSI

https://jira-pro.its.hpecorp.net:8443/browse/CASMNET-1665